example/SelColumn.qml

```diff
@@ -21,7 +21,7 @@ Widget {
             col.clear_sel if col.clear_on_extern
             col.setValue(col.filter(x)) if !col.skip
             col.setValue(x)             if  col.skip
-            // Resets the scroll bar to the top.
+            # Resets the scroll bar to the top.
```

Comment:
Heh, yeah... I'm tempted to add in a transform within mruby-qml-parse or mruby-qml-spawn to make these two equivalent. I've made this sort of typo plenty of times in the past.